src/Build/Logging/FancyLogger/FancyLogger.cs

```diff
@@ -79,6 +85,16 @@ void eventSource_ProjectFinished(object sender, ProjectFinishedEventArgs e)
             // Update line
             node.Finished = true;
             node.Log();
+            // Update footer
+            FinishedProjects++;
+            if (StartedProjects > 0)
```

Comment:
Is StartedProjects ever not > 0 here?

Reply:
You are right. it should always be > 0. This was me trying to be over-careful. Will remove it as it is redundant.